content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,220 @@ We have deprecated the following cert-manager APIs:
+
+```yaml
+# values.yaml
+webhook:
+  serviceType: LoadBalancer
+  loadBalancerIP: 85.78.90.100
+  url:
+    host: 85.78.90.100
+```
+
+### Helm chart: Service labels
+
+The cert-manager controller Service
+[now](https://github.com/jetstack/cert-manager/pull/4009) supports custom labels
+using the top-level field in `values.yaml`:
+
+```yaml
+# values.yaml
+serviceLabels:
+  app: armada-api
+```
+
+This may be useful in conjunction with Prometheus' `labelmap`. For example, with
+the following sample Prometheus configuration:
+
+```yaml
+# prometheus.yaml
+- action: labelmap
+  regex: __meta_kubernetes_service_label_(.+)
+```
+
+With the above example, the source label
+`__meta_kubernetes_service_label_app='armada-api'` becomes the new label
+`app='armada-api'` when metrics related to this Service are scraped.
+
+## Akamai issuer and Open Edgegrid EdgeDNS v2
+
+The Akamai issuer has been
+[updated](https://github.com/jetstack/cert-manager/pull/4007) to use Open
+Edgegrid EdgeDNS v2 API.
+
+# Bug Fixes
+
+- The default certificate renewal duration of 30 days was clashing with the
+  duration of certificates issued by the Vault PKI. All Certificates [are
+  now](https://github.com/jetstack/cert-manager/pull/4092) renewed 2/3 through
+  the duration unless a custom renew period is specified with the setting
+  `spec.renewBefore` on the Certificate.
+- The [RFC2136](https://cert-manager.io/docs/configuration/acme/dns01/rfc2136/)
+  issuer is [now](https://github.com/jetstack/cert-manager/pull/3622) able to
+  handle DNS-01 challenges that map to multiple `TXT` records. This lets you
+  create Let's Encrypt certificates using RFC2136 with multiple DNS names.
+- The comparison function `PublicKeysEqual` is
+  [now](https://github.com/jetstack/cert-manager/pull/3914) correct for public
+  keys.
+- The upgrade from v1.3.0 to 1.4.0 with Helm and `--set installCRDs=true` is
+  [now](https://github.com/jetstack/cert-manager/pull/3882) possible. This issue
+  was due to [a Helm
+  bug](https://github.com/helm/helm/issues/5806#issuecomment-788116838) with the
+  `minimum` field on the CRDs.
+- The ACME issuer [now](https://github.com/jetstack/cert-manager/pull/3866)
+  works correctly with Certificates that have a long name (52 characters or
+  more). These Certificates would not get renewed due to non-unique `Order`
+  names being generated.
+- Orders that are used with a misbehaving ACME server should not get stuck
+  [anymore](https://github.com/jetstack/cert-manager/pull/3805). By misbehaving,
+  we mean an ACME server that would validate the authorizations before having
+  set the status of the order to "ready".
+- The internal signers [now](https://github.com/jetstack/cert-manager/pull/3878)
+  set the condition `Ready=False` with the reason `RequestDenied` when a
+  CertificateRequest is `Denied`. This is to keep the same behavior where a
+  terminal state of a CertificateRequest should have a `Ready` condition.
+
+# Honorable mentions
+
```

Comment:
I know I'm too straight laced, but I just don't think this is appropriate for the release notes. It would be appropriate for a blog post.
Mentioning a not-merged feature in the release notes is just going to confuse people I think.